newIDE/app/src/UI/SortableVirtualizedItemList/index.js

```diff
@@ -96,10 +96,10 @@ class ItemsList extends React.Component<ItemsListProps, *> {
               getThumbnail={getThumbnail ? () => getThumbnail(item) : undefined}
               selected={item === selectedItem}
               onItemSelected={this.props.onItemSelected}
+              errorStatus={erroredItems ? erroredItems[item.getName()] : 0}
```

Comment:
`erroredItems ? erroredItems[item.getName()] : ''` if I'm not mistaken

Reply:
thanks :D sorry I missed it

Reply:
Also, if `erroredItems` does not contain `item.getName()`, you'll get undefined.
To be 100% correct, you should do:
`erroredItems ? erroredItems[item.getName()] || '' : ''}`

(which I admit, start to be a bit cryptic, but when you're used to it, this translate to "no error if erroredItems does not exist, and no error if the item is not found in these erroredItems".)

Reply:
good catch! 👍 added to the pr. Sorry I missed it :)